app/models/member.rb

```diff
@@ -68,8 +68,8 @@ def self.after(member)
     where("created_at > ?", member.created_at)
   end
 
-  def full_name
-    "#{first_name} #{last_name}"
+  def display_name
+    "#{first_name} #{last_name}".titleize
```

Comment:
this is where the magic is happening

Reply:
If the member is the _primary_ member should we use "You" instead of their name? I've seen in a story or two where it's been split in that way - "You" versus "Jane/John Doe".

Reply:
good idea / point!

Reply:
looking through the diff, changing `Joe Shmoe` to `you` would end up creating a product change rather than be a refactor of how we are displaying names, so for now I think it probably makes sense to keep as-is but note that we may change this in the future (and when that happens, having this method will hopfefully make it an easy change!)